.github/workflows/debian.yml

```diff
@@ -0,0 +1,233 @@
+name: Create debian package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  build_debian:
+
+    runs-on: ubuntu-latest
+
+    name: Build Debian Package
+
+    outputs:
+      package_name: ${{ steps.get-package-name.outputs.package_name }}
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        path: gpdb1
+
+
+    - uses: actions/checkout@v4
+      with:
+        repository: open-gpdb/gpdb_deb
+        ref: 6_jammy_github
+        path: gpdb/greenplum-build
+
+    - name: prepare environment
+      run: |
+        set -ex
+
+        mv gpdb1/* gpdb/
+        mv gpdb1/.[!.]* gpdb/
+
+        ls -al .
+        ls -al ./*
+
+        du -h .
+        cd gpdb && git branch; cd ..
+
+        rm -fr gpdb/gpcontrib/yezzey 
+        git clone https://github.com/open-gpdb/yezzey.git gpdb/gpcontrib/yezzey -b v1.8_opengpdb
```

Comment:
why?

Why it is not git-submoule? Whats wrong with `open-gpdb/gpcontrib/yezzey`?

Reply:
+1, removing yezzey is CI step, not release

Reply:
Added usage of git-submodule